recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+    name: {{ name|lower }}
```

Comment:
From aesthetic purposes, can you please fix `meta.yaml` such that each nested row is 2 spaces in, instead of 4, e.g.:
```
extra:
  recipe-maintainers:
    - leej3
```
?